source/process/marketing-guidelines.rst

```diff
@@ -63,3 +63,23 @@ Guidelines: 12 tips for sparkling content
  10. **Edit ruthlessly**. Use the first draft to get your thoughts together. Then leave it for a few hours (or even days). Shorten, delete, and rewrite anything that does not add to the meaning. Run a spelling and grammar check.
  11. **Ask a colleague**. Still not 100% happy? A colleague should review the text and tell you if anything is unclear or if you could express yourself better. They might offer some useful fresh insights.
  12. **Now, relax**. If you are in the wrong frame of mind you will either get writer's block or your writing will be stiff and unnatural. Take your time. Remove any distractions. Keep things simple. 
+
+
+Twitter Guidelines 
+==============================================================
+
+Composing Tweets: 
+
+1. **Avoid acronyms** - Avoid acronyms when possible. For example, say "pull request" instead of PR", since only a subset of Tweet readers will be active GitHub users. 
+2. **Use the active voice** - Avoid "has", "was", "have been" when possible. For example, instead of "Hackfest has started!" say "Hackfest starts now!" 
+3. **Include zero to one links, close to the end"** - Include no more than one link per tweet, and place it close to the end of the tweet. 
```

Comment:
While I agree what's there isn't great, I don't know if `No more than one link, placed near the end` is much better.